physical/consul.go

```diff
@@ -30,22 +30,24 @@ const (
 	// be executed too close to the TTL check timeout
 	checkMinBuffer = 100 * time.Millisecond
 
+	// consulRetryInterval specifies the retry duration to use when an
+	// API call to the Consul agent fails.
+	consulRetryInterval = 1 * time.Second
+
 	// defaultCheckTimeout changes the timeout of TTL checks
 	defaultCheckTimeout = 5 * time.Second
 
-	// defaultCheckInterval specifies the default interval used to send
-	// checks
-	defaultCheckInterval = 4 * time.Second
-
 	// defaultServiceName is the default Consul service name used when
 	// advertising a Vault instance.
 	defaultServiceName = "vault"
 
-	// registrationRetryInterval specifies the retry duration to use when
-	// a registration to the Consul agent fails.
-	registrationRetryInterval = 1 * time.Second
+	// reconcileTimeout is how often Vault should query Consul to detect
+	// and fix any state drift.
+	reconcileTimeout = 60 * time.Second
 )
 
+type notifyEvent struct{}
```

Comment:
This gives an impression that we are sending some info down the channel.
It can simply be like this:
`notifyActiveCh chan notifyEvent` --> `notifyActiveCh chan struct{}`
`case c.notifyActiveCh <- notifyEvent{}` --> `case c.notifyActiveCh <- struct{}{}`